edx_proctoring/backends/rest.py

```diff
@@ -122,7 +122,7 @@ def get_javascript(self):
                 u'Webpack stats file corresponding to WebWorkers not found: {}'
                 .format(str(err))
             )
-        return js_url
+        return js_url + u'?raw'
```

Comment:
from here: https://github.com/edx/edx-proctoring/compare/matthugs/we-may-need-this-to-integrate-other-proctoring-backends

Reply:
I'm not sure I want to ship this without @davestgermain's input on it; the goal of this is to hit https://github.com/edx/edx-platform/blob/master/common/djangoapps/static_replace/__init__.py#L176 and avoid static rewriting of this URL by automated static URL rewriting infrastructure.